### ISSUE TITLE
Patch multiple objects

### ISSUE THREAD

**Issue by glebmikulko** [wopian/kitsu#336]
I am wondering how can I patch multiple objects? In what format should I pass modified data?

**Comment by glebmikulko** [comment]
For example, I have multiple `setting`s and want to patch them on a backend server. I tried to do the following: 
```
kitsu.patch('setting', {
      id: '1,2,3,4,5,6,7',
      data: { id: '1,2,3,4,5,6,7', settings: outer_variable },
    })
```
but got wrong nesting structure on the backend:
```
 {"type"=>"setting",
   "id"=>"1,2,3,4,5,6,7",
   "relationships"=>
    {"data"=>
      {"data"=>
        {"id"=>"1,2,3,4,5,6,7",
         "settings"=>
          [{"id"=>"1", "type"=>"setting", "store"=>{"id"=>"1", "type"=>"store"}, "weekDay"=>1, "time"=>900},
           {"id"=>"2", "type"=>"setting", "store"=>{"id"=>"1", "type"=>"store"}, "weekDay"=>2, "time"=>900},
           {"id"=>"3", "type"=>"setting", "store"=>{"id"=>"1", "type"=>"store"}, "weekDay"=>3, "time"=>900},
           {"id"=>"4", "type"=>"setting", "store"=>{"id"=>"1", "type"=>"store"}, "weekDay"=>4, "time"=>900},
           {"id"=>"5", "type"=>"setting", "store"=>{"id"=>"1", "type"=>"store"}, "weekDay"=>5, "time"=>900},
           {"id"=>"6", "type"=>"setting", "store"=>{"id"=>"1", "type"=>"store"}, "weekDay"=>6, "time"=>900},
           {"id"=>"7", "type"=>"setting", "store"=>{"id"=>"1", "type"=>"store"}, "weekDay"=>7, "time"=>900}],
         "type"=>"data"}}}}
```
(double `data` nesting in `realtionships`)

**Comment by glebmikulko** [comment]
@wopian :weary:

**Comment by wopian** [comment]
Hey, sorry I was busy with university work and this issue fell through the gaps while I only briefly checked GitHub to merge automated dependency updates.

---

I don't think this is currently possible with this package. 

I've checked the [JSON:API 1.0 specification](https://jsonapi.org/format/) documentation and can't find anything on it, so it's possibly a behaviour other JSON:API server/client implementations have added on top of the standard?

**Comment by glebmikulko** [comment]
Seems like it's true. It's an [experimental](http://springbot.github.io/json-api/extensions/bulk/) feature at the moment. It's a pity

**Comment by wopian** [comment]
Bulk Extension support for `PATCH` and `POST` added in 920ece3

**Comment by wopian** [comment]
Bulk Extension support for DELETE added in f793988.

**Comment by wopian** [comment]
Syntax for `PATCH` and `POST` are the same as with objects, just wrapped in an array:

```js
api.post('users', [ 
	{ name: 'wopian' },
	{ name: 'glebmikulko' } 
])
api.patch('users', [ 
	{ id: '1', name: 'wopian' },
	{ id: '2', name: 'glebmikulko' } 
])
```

Syntax for `DELETE` is simply:

```js
api.delete('users', [ 1, 2 ])
```

**Comment by wopian** [comment]
Released in `9.1.0`

**Comment by glebmikulko** [comment]
Totally cool, thank you. I'll give it a shot :1st_place_medal:
